test/libsolidity/lsp/references/enum.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+enum Color {
+//   ^ @EnumDef
+//   ^^^^^ @ColorType0
+    Red,
+//  ^ @RedDef
+//  ^^^ @Red1
+    Green
+}
+
+contract MyContract
```

Comment:
Can you add something to test finding references in identifier paths? E.g. a modifier invocation or an imported function being used in `using for`? Especially when the reference you want to find is not the last part of the identifier path.